test/ExportQueryExecutionTreesTest.cpp

```diff
@@ -636,6 +658,52 @@ TEST(ExportQueryExecutionTrees, LiteralWithDatatype) {
   runConstructQueryTestCase(testCaseConstruct);
 }
 
+// ____________________________________________________________________________
+TEST(ExportQueryExecutionTrees, LiteralCornercases) {
+  auto runTestCaseWithLiteral = [](std::string literal) {
+    const std::string innerValue = literal.substr(1, literal.rfind("\"") - 1);
+    const std::string suffixValue = literal.substr(literal.rfind("\"") + 1);
+    std::string kg = absl::StrCat("<s> <p> ", literal);
```

Comment:
Is this code by you, or is this shown because you haven't merged in the current master?